snmp/datadog_checks/snmp/data/default_profiles/citrix-netscaler-sdx.yaml

```diff
@@ -0,0 +1,350 @@
+          6: ipv6z
+          16: dns
+      - column:
+          name: netscaler.sdx.nsNetmask
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.8
+          format: ip_address
+        tag: netscaler_sdx_ns_netmask
+      - column:
+          name: netscaler.sdx.nsGatewayType
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.9
+        tag: netscaler_sdx_ns_gateway_type
+        mapping:
+          0: unknown
+          1: ipv4
+          2: ipv6
+          3: ipv4z
+          6: ipv6z
+          16: dns
+      - column:
+          name: netscaler.sdx.nsGateway
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.10
+          format: ip_address
+        tag: netscaler_sdx_ns_gateway
+      - column:
+          name: netscaler.sdx.nsHostname
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.11
+        tag: netscaler_sdx_ns_hostname
+      - column:
+          name: netscaler.sdx.nsDescription
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.12
+        tag: netscaler_sdx_ns_description
+      - column:
+          name: netscaler.sdx.nsVersion
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.13
+        tag: netscaler_sdx_ns_version
+      - column:
+          name: netscaler.sdx.nsVmDescription
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.30
+        tag: netscaler_sdx_ns_vm_description
+      - column:
+          name: netscaler.sdx.nsThroughput
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.31
+        tag: netscaler_sdx_ns_throughput
+      - column:
+          name: netscaler.sdx.nsHaIPAddressType
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.42
+        tag: netscaler_sdx_ns_ha_ip_address_type
+        mapping:
+          0: unknown
+          1: ipv4
+          2: ipv6
+          3: ipv4z
+          6: ipv6z
+          16: dns
+      - column:
+          name: netscaler.sdx.nsHaIPAddress
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.43
+          format: ip_address
+        tag: netscaler_sdx_ns_ha_ip_address
+      - column:
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.15
+          name: netscaler.sdx.nsInstanceState
+        tag: netscaler_sdx_ns_instance_state
+      - column:
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.18
+          name: netscaler.sdx.nsVmState
+        tag: netscaler_sdx_ns_vm_state
+      - column:
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.41
+          name: netscaler.sdx.nsHaMasterState
+        tag: netscaler_sdx_ns_ha_master_state
+      - column:
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.44
+          name: netscaler.sdx.nsNodeState
+        tag: netscaler_sdx_ns_node_state
+      - column:
+          OID: 1.3.6.1.4.1.5951.6.3.2.1.45
+          name: netscaler.sdx.nsHaSync
+        tag: netscaler_sdx_ns_ha_sync
+metric_tags:
+  - OID: 1.3.6.1.4.1.5951.6.2.2.0
+    symbol: systemProduct
+    tag: netscaler_sdx_system_product
+  - OID: 1.3.6.1.4.1.5951.6.2.4.0
+    symbol: systemSvmIPAddressType
+    tag: netscaler_sdx_system_svm_ip_address_type
+    # mapping:
+    #       0: unknown
+    #       1: ipv4
+    #       2: ipv6
+    #       3: ipv4z
+    #       6: ipv6z
+    #       16: dns
```

Comment:
Hum, don't we need those mapping?

The type of `systemSvmIPAddressType` seems to be `InetAddressType`:

```
InetAddressType ::= TEXTUAL-CONVENTION
    STATUS      current
    SYNTAX       INTEGER {
                     unknown(0),
                     ipv4(1),
                     ipv6(2),
                     ipv4z(3),
                     ipv6z(4),
                     dns(16)
                 }

```

without the mapping, the tags will be send as integer value, which is not very readable.

Reply:
I kept these commented out because they're metric tags at the top level of the device and I wasn't sure if they'd work. The tooling didn't add these mappings but to me it would make sense to have them. Will the mapping work on tags at this level of the profile?

If the tags aren't causing any problems or cost, wouldn't it be better to have them?